lib/config/fileconf.go

```diff
@@ -991,6 +991,15 @@ type SSH struct {
 	// X11 is used to configure X11 forwarding settings
 	X11 *X11 `yaml:"x11,omitempty"`
 
+	// MaybeFileCopying enables or disables remote file operations via SCP/SFTP.
+	// We're using a pointer-to-bool here because the system default is to allow
+	// TCP forwarding, we need to distinguish between an unset value and a false
+	// value so we can an override unset value with `true`.
+	//
+	// Don't read this value directly: call the FileCopying method
+	// instead.
+	MaybeFileCopying *bool `yaml:"file_copying,omitempty"`
```

Comment:
I might suggest a name that makes it clear this is for SSH, otherwise it may get confused with desktop access where file sharing is a completely separate option.

Reply:
I can do that, I thought it was clear enough though as the field is in the `SSH` struct